src/pepr/operator/crd/index.ts

```diff
@@ -20,17 +20,19 @@ export {
 } from "./generated/exemption-v1alpha1";
 
 export {
-  VirtualService as IstioVirtualService,
-  HTTPRoute as IstioHTTPRoute,
   HTTP as IstioHTTP,
+  HTTPRoute as IstioHTTPRoute,
+  VirtualService as IstioVirtualService,
 } from "./generated/istio/virtualservice-v1beta1";
 
 export {
-  ServiceEntry as IstioServiceEntry,
-  Location as IstioLocation,
-  Resolution as IstioResolution,
   Endpoint as IstioEndpoint,
+  Location as IstioLocation,
   Port as IstioPort,
+  Resolution as IstioResolution,
+  ServiceEntry as IstioServiceEntry,
 } from "./generated/istio/serviceentry-v1beta1";
 
+export { Action, AuthorizationPolicy } from "./generated/istio/authorizationpolicy-v1beta1";
+export { RequestAuthentication } from "./generated/istio/requestauthentication-v1";
```

Comment:
nit: maybe `export as` with a prefix of `Istio` to align with other istio resources?